dbt_subprojects/dex/models/trades/base/platforms/otsea_base_base_trades.sql

```diff
@@ -0,0 +1,90 @@
+{{ config(
+    schema = 'otsea_base',
+    alias = 'base_trades',
+    partition_by = ['block_month'],
+    materialized = 'incremental',
+    file_format = 'delta',
+    incremental_strategy = 'merge',
+    unique_key = ['block_date', 'blockchain', 'project', 'version', 'tx_hash', 'evt_index']
```

Comment:
We need to make sure the `unique_key` match the key in the uniqueness test:
```suggestion
    unique_key = ['tx_hash', 'evt_index']
```